### DISCUSSION TITLE
Is there a convenient way to have a collection pull in latest git updates?

### DISCUSSION THREAD

**Discussion by stray-signals** [usebruno/bruno#2233] [Q&A]
The organization I am with is experimenting with replacing postman with Bruno, but something that has come up that I am wondering about is there a convenient way to have the Bruno UI sync changes from a master branch. We have non-technical users that would be using this to test changes before they go live, and it would be helpful for them to always have the latest updates from a repo.

Edit:
So I realized after this that in the golden edition a guy ui is unlocked, however that gitUI does not seem to be able to see the latest updates until they are already pulled down, and does not seem to have any ability to pull or push code in the Bruno UI.
While not a breaking feature in any way, part of my use case is to allow non-technical users to test out APIs and without a convenient way to do so it may slow down adoption for some use cases.

**Comment by helloanoop** [comment]
We've made a lot of improvements with Git Integration recently.
Please checkout the latest version - `1.21.0`
 
The one feature still pending is the Git pull and push functionality.
We are targeting to have this released on Tuesday, July 23rd.
